### PR TITLE
#147: restyle story tags list

### LINKED ISSUE #147: Tap targets are not sized appropriately
Run a Lighthouse test on a story page. Need to ensure clickable elements are at least 48x48 pixels

doc: https://web.dev/tap-targets/?utm_source=lighthouse&utm_medium=devtools

### PR DESCRIPTION
**Pull request by brandonhundt**
Closes #147 

- The tags and categories list at the end of stories didn't have appropriate sizing (48x48) so I redesigned them a bit here. 

## To Review

- [ ] Use the Preview link located in the Now comment below.

> ...or...

- [ ] Checkout Branch.
- [ ] Run `yarn`.
- [ ] Run `yarn dev:start`.
- [ ] Go to [localhost:3000](https://localhost:3000).

> ...then...

- [ ] Go to any story and view the category styling (desktop and mobile)
- [ ] Run Lighthouse on the story, ensure tap target size issues have all been resolved.

### CODE REVIEW

File: components/Tags/Tags.styles.ts [hunk 1]
```diff
@@ -8,42 +8,27 @@ import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
...
+      padding: '0',
       fontWeight: theme.typography.fontWeightRegular,
       '&::after': {
         content: '":\x20"'
       }
     },
     link: {
-      fontWeight: theme.typography.fontWeightBold
+      fontWeight: theme.typography.fontWeightBold,
+      margin: '0 2px',
```
Comment (rpeterman-gp) [inline components/Tags/Tags.styles.ts:27]: Spacing of buttons should be controlled on the parent using `gap` property. Value should be in rem's calculated using theme helpers, `gap: theme.typography.pxToRem(theme.spacing(1))`.

File: components/Tags/Tags.styles.ts [hunk 2]
```diff
@@ -8,42 +8,27 @@ import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
 export const tagsStyles = makeStyles((theme: Theme) =>
   createStyles({
     root: {
-      marginTop: '1rem',
-      marginBottom: '1rem',
+      display: 'flex',
+      alignItems: 'center',
+      flexWrap: 'wrap',
+      margin: '.5rem 0',
```
Comment (rpeterman-gp) [inline components/Tags/Tags.styles.ts:14]: Spacing values should be calculated with theme helpers, ``margin: `${theme.typography.pxToRem(theme.spacing(1))} 0`, ``

File: components/Tags/Tags.styles.ts [hunk 3]
```diff
@@ -8,42 +8,27 @@ import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
...
       color: theme.palette.grey[700]
     },
     label: {
-      display: 'inline',
+      margin: '0',
+      padding: '0',
```
Comment (rpeterman-gp) [inline components/Tags/Tags.styles.ts:19]: This padding reset is unnecessary.

**Review by rpeterman-gp — CHANGES_REQUESTED** [review]
Just a couple of changes to better leverage the theme when styling layout.

File: components/Tags/Tags.styles.ts (hunk 1 above)
Comment (brandonhundt) [reply components/Tags/Tags.styles.ts]: Fixed!

File: components/Tags/Tags.styles.ts (hunk 3 above)
Comment (brandonhundt) [reply components/Tags/Tags.styles.ts]: Fixed!

File: components/Tags/Tags.styles.ts (hunk 2 above)
Comment (brandonhundt) [reply components/Tags/Tags.styles.ts]: Fixed!

File: components/Tags/Tags.styles.ts [hunk 4]
```diff
@@ -11,20 +11,19 @@ export const tagsStyles = makeStyles((theme: Theme) =>
...
-      padding: '0',
       fontWeight: theme.typography.fontWeightRegular,
       '&::after': {
         content: '":\x20"'
       }
     },
     link: {
       fontWeight: theme.typography.fontWeightBold,
-      margin: '0 2px',
+      gap: theme.typography.pxToRem(theme.spacing(1)),
```
Comment (rpeterman-gp) [inline components/Tags/Tags.styles.ts:26]: The `gap` prop should be on the flex container, `root`.

**Review by rpeterman-gp — CHANGES_REQUESTED** [review]

**Review by rpeterman-gp — APPROVED** [review]

**Pull request merged**